src/tests/miscellaneous/mock-adb/common-adb-configs.js

```diff
@@ -23,24 +66,25 @@ function workingDeviceCommands(deviceIds, port) {
     }
 
     for (const id of deviceIds) {
-        output[`-s ${id} devices`] = cloneDeep(output.devices);
+        output[`-s ${id} ` + devicesCommandMatch] = cloneDeep(output.devices);
```

Comment:
we can likely use interpolation (like the id variable) for the Match variables too; this might be nice so we're not using both ways (interpolation and concatenation) to build the string

something like `-s ${id} ${devicesCommandMatch}`

Reply:
Fixed